test/test-applications/integrations/Samples.GraphQL/Program.cs

```diff
@@ -31,7 +31,7 @@ public static void Main(string[] args)
             var logger = host.Services.GetRequiredService<ILogger<Program>>();
             logger.LogInformation($"Instrumentation.ProfilerAttached = {Instrumentation.ProfilerAttached}");
 
-            var prefixes = new[] { "COR_", "CORECLR_", "DD_", "DATADOG_" };
+            var prefixes = new[] { "COR_", "CORECLR_", "OTEL_", "DATADOG_" };
```

Comment:
The `DD_` prefix was replaced here, should the `DATADOG_` prefix be replaced/removed here too?

Reply:
As far I as understands _DATADOG is a legacy prefix and shouldn't affect anything, so it can possibly be removed, but I didn't have guts to do this.